scst-store/tls-configuration.hbs.md

```diff
@@ -0,0 +1,53 @@
+
+# Setting up TLS in Metadata Store
+
+###  <a id="Setting up tls namespace and secretName"></a>Setting up tls namespace and secretName
```

Comment:
I don't think you can set the `id` to a string with spaces in it. I think we typically set it to something with some delimiter like `-`.